plugins/foreman_cockpit/1.0/index.md

```diff
@@ -42,6 +42,10 @@ To install Foreman Cockpit, run the following:
 
       # yum install tfm-rubygem-foreman_cockpit
 
+or this if your Foreman version is 1.9 or older
```

Comment:
missing bullet point at the start of these lines I think, the formatting looks inconsistent with the others